Program.cs

```diff
@@ -0,0 +1,31 @@
+using System.Configuration;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.EntityFrameworkCore.InMemory;
```

Comment:
can you please explain the purpose of using InMemory here ?

Reply:
First I was running the code without database so using local memory for saving data so it was being used for that purpose.